attackers/create_update_request.py

```diff
@@ -0,0 +1,13 @@
+import ast
+
+
+def update_request():

```

Comment:
`update_request` sounds like you give it a request and it updates it.  
consider:  
```python
def create_update_request(path: Path) -> Request:
```